### PR TITLE
Relax click version upper bound to 8.0

### PR DESCRIPTION
**Pull request by mariobriggs**
Requested in https://github.com/encode/uvicorn/issues/1016

Click 8.0 has security patches

### CODE REVIEW

**Comment by mariobriggs** [comment]
@Kludex / Others click 8.0.0 has released. Do u see any blockers to upgrading uvicorn to it ? 

thanks in advance

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def get_packages(package):
 env_marker_below_38 = "python_version < '3.8'"
 
 minimal_requirements = [
-    "click==7.*",
+    "click==8.*",
```
Comment (euri10) [inline setup.py]: Why not relax just the upper bound ?

Comment (mariobriggs) [reply setup.py]: @euri10 done. I have made it >=7.*

Comment (mariobriggs) [reply setup.py]: image

**Review by euri10 — APPROVED** [review]

**Comment by euri10** [comment]
LGTM, can you please fix the CI ?
Seems like running the lint script might be enough

**Review by euri10 — CHANGES_REQUESTED** [review]
Linting seems off for the CI checks, can you please run it ?
it's in scripts/lint

**Comment by mariobriggs** [comment]
> Linting seems off for the CI checks, can you please run it ?
> it's in scripts/lint

DONE

**Comment by euri10** [comment]
thanks @mariobriggs I went on merging https://github.com/encode/uvicorn/pull/1042 just before !

**Pull request closed without merging**